domenetjenester/registerinnhenting/src/main/java/no/nav/foreldrepenger/domene/registerinnhenting/impl/startpunkt/StartpunktUtlederInntektsmelding.java

```diff
@@ -90,8 +90,9 @@ private List<Inntektsmelding> hentInntektsmeldingerFraGittGrunnlag(Optional<Innt
             return StartpunktType.KONTROLLER_ARBEIDSFORHOLD;
         }
 
-        if ( FagsakYtelseType.SVANGERSKAPSPENGER.equals(ref.fagsakYtelseType()) && nyeIm.stream().anyMatch(nyIm -> nyttArbForholdForEksisterendeArbgiver(nyIm, gamleIm))) {
-            return StartpunktType.INNGANGSVILKÅR_OPPLYSNINGSPLIKT;
+        if ( FagsakYtelseType.SVANGERSKAPSPENGER.equals(ref.fagsakYtelseType()) && (nyeIm.stream().anyMatch(nyIm -> nyttArbForholdForEksisterendeArbgiver(nyIm, gamleIm))
+            || nyeIm.stream().anyMatch(nyIm -> endringIArbeidsforholdIdForSammeArbgiver(nyIm, gamleIm)))) {
+            return StartpunktType.KONTROLLER_FAKTA;
```

Comment:
Den der kan gi evig løkke. 
Du må legge inn det som skjer i KOFAK-steg i Endringskontroller med en if(SVP)

Reply:
Nytt steg for å gjøre vurdering av tilrettelegginger er implementert. Har også lagt inn opprettelse av SVP_TILRETELEGGINGER aksjonspunkt her, og fjernet kode vedr tilrettelgginger fra fakta steg.